lib/auth.coffee

```diff
@@ -281,6 +281,52 @@ getAuth = (deps, opts) ->
 		.get('id')
 		.asCallback(callback)
 
+	###*
+	# @summary Get current logged in user's personal organization id
+	# @name getPersonalOrganizationId
+	# @public
+	# @function
+	# @memberof balena.auth
+	#
+	# @description This will only work if you used {@link balena.auth.login} to log in.
+	#
+	# @fulfil {Number} - user's personal organization id
+	# @returns {Promise}
+	#
+	# @example
+	# balena.auth.getPersonalOrganizationId().then(function(organizationId) {
+	# 	console.log(organizationId);
+	# });
+	#
+	# @example
+	# balena.auth.getPersonalOrganizationId(function(error, organizationId) {
+	# 	if (error) throw error;
+	# 	console.log(organizationId);
+	# });
+	###
+	exports.getPersonalOrganizationId = (callback) ->
```

Comment:
Should we have a `getPersonalOrganization` method that accepts a `$select` instead, so the caller can decide which fields they need (in this case just ID)? This applies to all methods we expose from the SDK, just thinking which approach would be better here, no action required. We can then have some commonly used cases (like `getPersonalOrganizationId` exposed as methods that simply utilize this more general `getPersonalOrganization`.

Reply:
@sradevski  I was thinking about this, since I we also need this in the UI, but that doesn't sound like that should be part of the `.auth` namespace.
If we are going to add it, then it probably deserves a proper namespace in `models`, which I'm not sure we should do, since it will not work with the open-balena-api :thinking:
On the other hand the `getPersonalOrganizationId()` just returns an id which allows us to safely cache it along with the user details.
How would you feel about having such a method in the `.auth` namespace?

Reply:
@thgreasi I agree that it doesn't belong in the `auth` namespace, I was thinking more like an `organization` model that `getPersonalOrganization` and `getPersonalOrganizationId` that the `auth` namespace can use. Anyway, just thinking out loud, I am fine without doing that for this PR

Reply:
Always glad to hear your thoughts on these :)
We can always add it later if it ends up being a crucial method.